internal/loadtest/events/GenericEvent/events.go

```diff
@@ -0,0 +1,206 @@
+package GenericEvent
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	cev2 "github.com/cloudevents/sdk-go/v2"
+
+	"github.com/kyma-project/eventing-tools/internal/loadtest/events/payload"
+	"github.com/kyma-project/eventing-tools/internal/tree"
+)
+
+type Event struct {
+	source    string
+	version   string
+	EventName string
+	eps       int
+	Starttime string
```

Comment:
What is the reason for `EventName` and `Starttime` being exported?

Reply:
changed